--- cvxopt-1.1.3/src/setup.py.orig	2011-12-03 20:46:13.974704473 -0200
+++ cvxopt-1.1.3/src/setup.py	2011-12-03 20:50:02.271777389 -0200
@@ -106,7 +106,7 @@ umfpack = Extension('umfpack',
         'C/SuiteSparse/UFconfig' ],
     library_dirs = [ ATLAS_LIB_DIR ],
     define_macros = MACROS,
-    libraries = [ 'blas', 'lapack'],
+    libraries = [ 'blas', 'lapack', 'm'],
     sources = [ 'C/umfpack.c',
         'C/SuiteSparse/UMFPACK/Source/umfpack_global.c',
         'C/SuiteSparse/UMFPACK/Source/umfpack_tictoc.c' ] +
@@ -118,7 +118,7 @@ if sys.maxint > 2**31: MACROS += [('DLON
 
 cholmod = Extension('cholmod',
     library_dirs = [ ATLAS_LIB_DIR ],
-    libraries = ['lapack', 'blas'],
+    libraries = ['lapack', 'blas', 'm' ],
     include_dirs = [ 'C/SuiteSparse/CHOLMOD/Include', 
         'C/SuiteSparse/COLAMD', 'C/SuiteSparse/AMD/Include', 
         'C/SuiteSparse/UFconfig', 'C/SuiteSparse/COLAMD/Include' ],
@@ -134,12 +134,13 @@ cholmod = Extension('cholmod',
         glob('C/SuiteSparse/CHOLMOD/Supernodal/c*.c') )
 
 amd = Extension('amd', 
+    libraries = [ 'm' ],
     include_dirs = [ 'C/SuiteSparse/AMD/Include', 
         'C/SuiteSparse/UFconfig' ],
     define_macros = MACROS,
     sources = [ 'C/amd.c' ] + glob('C/SuiteSparse/AMD/Source/*.c') )
 
-misc_solvers = Extension('misc_solvers', libraries = ['lapack', 'blas'],
+misc_solvers = Extension('misc_solvers', libraries = ['lapack', 'blas', 'm' ],
     library_dirs = [ ATLAS_LIB_DIR ],
     define_macros = MACROS,
     sources = ['C/misc_solvers.c'] )
