--- cvxopt-1.1.6/setup.py.orig	2013-12-04 19:44:01.108950839 -0200
+++ cvxopt-1.1.6/setup.py	2013-12-04 19:45:16.046947756 -0200
@@ -7,6 +7,7 @@ BLAS_LIB_DIR = '/usr/lib'
 # Default names of BLAS and LAPACK libraries
 BLAS_LIB = ['blas']
 LAPACK_LIB = ['lapack']
+M_LIB = ['m']
 BLAS_EXTRA_LINK_ARGS = []
 
 # Set environment variable BLAS_NOUNDERSCORES=1 if your BLAS/LAPACK do
@@ -84,7 +85,7 @@ if BUILD_GLPK:
     extmods += [glpk];
 
 if BUILD_DSDP:
-    dsdp = Extension('dsdp', libraries = ['dsdp'] + LAPACK_LIB + BLAS_LIB,
+    dsdp = Extension('dsdp', libraries = ['dsdp'] + LAPACK_LIB + BLAS_LIB + M_LIB,
         include_dirs = [ DSDP_INC_DIR ],
         library_dirs = [ DSDP_LIB_DIR, BLAS_LIB_DIR ],
         extra_link_args = BLAS_EXTRA_LINK_ARGS,
@@ -93,19 +94,19 @@ if BUILD_DSDP:
 
 # Required modules
 
-base = Extension('base', libraries = ['m'] + LAPACK_LIB + BLAS_LIB,
+base = Extension('base', libraries = ['m'] + LAPACK_LIB + BLAS_LIB + M_LIB,
     library_dirs = [ BLAS_LIB_DIR ],
     define_macros = MACROS,
     extra_link_args = BLAS_EXTRA_LINK_ARGS,
     sources = ['src/C/base.c','src/C/dense.c','src/C/sparse.c']) 
 
-blas = Extension('blas', libraries = BLAS_LIB,
+blas = Extension('blas', libraries = BLAS_LIB + M_LIB,
     library_dirs = [ BLAS_LIB_DIR ],
     define_macros = MACROS,
     extra_link_args = BLAS_EXTRA_LINK_ARGS,
     sources = ['src/C/blas.c'] )
 
-lapack = Extension('lapack', libraries = LAPACK_LIB + BLAS_LIB,
+lapack = Extension('lapack', libraries = LAPACK_LIB + BLAS_LIB + M_LIB,
     library_dirs = [ BLAS_LIB_DIR ],
     define_macros = MACROS,
     extra_link_args = BLAS_EXTRA_LINK_ARGS,
@@ -118,7 +119,7 @@ umfpack = Extension('umfpack',
         'src/C/SuiteSparse/SuiteSparse_config' ],
     library_dirs = [ BLAS_LIB_DIR ],
     define_macros = MACROS + [('NTIMER', '1'), ('NCHOLMOD', '1')],
-    libraries = LAPACK_LIB + BLAS_LIB,
+    libraries = LAPACK_LIB + BLAS_LIB + M_LIB,
     extra_compile_args = ['-Wno-unknown-pragmas'],
     extra_link_args = BLAS_EXTRA_LINK_ARGS,
     sources = [ 'src/C/umfpack.c',
@@ -133,7 +134,7 @@ if sys.maxsize > 2**31: MACROS += [('DLO
 
 cholmod = Extension('cholmod',
     library_dirs = [ BLAS_LIB_DIR ],
-    libraries = LAPACK_LIB + BLAS_LIB,
+    libraries = LAPACK_LIB + BLAS_LIB + M_LIB,
     include_dirs = [ 'src/C/SuiteSparse/CHOLMOD/Include', 
         'src/C/SuiteSparse/COLAMD', 
         'src/C/SuiteSparse/AMD/Include', 
@@ -153,13 +154,14 @@ cholmod = Extension('cholmod',
         glob('src/C/SuiteSparse/CHOLMOD/Supernodal/c*.c') )
 
 amd = Extension('amd', 
+    libraries = M_LIB,
     include_dirs = [ 'src/C/SuiteSparse/AMD/Include', 
         'src/C/SuiteSparse/SuiteSparse_config' ],
     define_macros = MACROS,
     sources = [ 'src/C/amd.c' ] + glob('src/C/SuiteSparse/AMD/Source/*.c') )
 
 misc_solvers = Extension('misc_solvers',
-    libraries = LAPACK_LIB + BLAS_LIB,
+    libraries = LAPACK_LIB + BLAS_LIB + M_LIB,
     library_dirs = [ BLAS_LIB_DIR ],
     define_macros = MACROS,
     extra_link_args = BLAS_EXTRA_LINK_ARGS,
