diff -up cvxopt-1.3.0/setup.py.1~ cvxopt-1.3.0/setup.py
--- cvxopt-1.3.0/setup.py.1~	2022-03-07 22:58:58.000000000 +0100
+++ cvxopt-1.3.0/setup.py	2022-11-06 13:51:13.921750297 +0100
@@ -4,11 +4,11 @@ import os, sys
 import versioneer
 
 # Modifiy this if BLAS and LAPACK libraries are not in /usr/lib.
-BLAS_LIB_DIR = '/usr/lib'
+BLAS_LIB_DIR = '/usr/lib/atlas'
 
 # Default names of BLAS and LAPACK libraries
-BLAS_LIB = ['blas']
-LAPACK_LIB = ['lapack']
+BLAS_LIB = ['satlas']
+LAPACK_LIB = ['satlas']
 BLAS_EXTRA_LINK_ARGS = []
 
 # Set environment variable BLAS_NOUNDERSCORES=1 if your BLAS/LAPACK do
@@ -17,7 +17,7 @@ BLAS_NOUNDERSCORES = False
 
 # Set to 1 if you are using the random number generators in the GNU
 # Scientific Library.
-BUILD_GSL = 0
+BUILD_GSL = 1
 
 # Directory containing libgsl (used only when BUILD_GSL = 1).
 GSL_LIB_DIR = '/usr/lib'
@@ -26,7 +26,7 @@ GSL_LIB_DIR = '/usr/lib'
 GSL_INC_DIR = '/usr/include/gsl'
 
 # Set to 1 if you are installing the fftw module.
-BUILD_FFTW = 0
+BUILD_FFTW = 1
 
 # Directory containing libfftw3 (used only when BUILD_FFTW = 1).
 FFTW_LIB_DIR = '/usr/lib'
@@ -35,7 +35,7 @@ FFTW_LIB_DIR = '/usr/lib'
 FFTW_INC_DIR = '/usr/include'
 
 # Set to 1 if you are installing the glpk module.
-BUILD_GLPK = 0
+BUILD_GLPK = 1
 
 # Directory containing libglpk (used only when BUILD_GLPK = 1).
 GLPK_LIB_DIR = '/usr/lib'
@@ -50,7 +50,7 @@ BUILD_DSDP = 0
 DSDP_LIB_DIR = '/usr/lib'
 
 # Directory containing dsdp5.h (used only when BUILD_DSDP = 1).
-DSDP_INC_DIR = '/usr/include/dsdp'
+DSDP_INC_DIR = '/usr/include/DSDP'
 
 # Guess SUITESPARSE_LIB_DIR and SUITESPARSE_INC_DIR
 if sys.platform.startswith("darwin"):
@@ -69,7 +69,7 @@ else:
     else:
         # Default
         SUITESPARSE_LIB_DIR = '/usr/lib'
-        SUITESPARSE_INC_DIR = '/usr/include'
+        SUITESPARSE_INC_DIR = '/usr/include/suitesparse'
 
 if sys.platform.startswith("win"):
     GSL_MACROS = [('GSL_DLL',''),('WIN32','')]
